ray-operator/test/sampleyaml/rayjob_test.go

```diff
@@ -0,0 +1,84 @@
+package sampleyaml
+
+import (
+	"testing"
+
+	. "github.com/onsi/gomega"
+
+	rayv1 "github.com/ray-project/kuberay/ray-operator/apis/ray/v1"
+	. "github.com/ray-project/kuberay/ray-operator/test/support"
+)
+
+func TestRayJob(t *testing.T) {
+	tests := []struct {
+		name string
+	}{
+		{
+			name: "ray-job.custom-head-svc.yaml",
+		},
+		{
+			name: "ray-job.modin.yaml",
+		},
+		{
+			name: "ray-job.resources.yaml",
+		},
+		{
+			name: "ray-job.sample.yaml",
+		},
+		{
+			name: "ray-job.shutdown.yaml",
+		},
+	}
+
+	for _, tt := range tests {
+		t.Run(tt.name, func(t *testing.T) {
+			test := With(t)
+			g := NewWithT(t)
+
+			namespace := test.NewTestNamespace()
+			test.StreamKubeRayOperatorLogs()
+			rayJobFromYaml := DeserializeRayJobSampleYAML(test, tt.name)
+			KubectlApplyYAML(test, tt.name, namespace.Name)
+
+			test.T().Logf("Waiting for RayJob %s/%s to be running", namespace.Name, rayJobFromYaml.Name)
+			g.Eventually(RayJob(test, namespace.Name, rayJobFromYaml.Name), TestTimeoutMedium).
+				Should(WithTransform(RayJobStatus, Equal(rayv1.JobStatusRunning)))
```

Comment:
Remove these lines because Running is a transition state, we don't need to check it here.